Dockerfile

```diff
@@ -163,12 +163,14 @@ COPY scripts/build_node_package.sh .
 RUN ./build_node_package.sh
 
 # node-package-builder-glibc
-FROM ubuntu${NODE_PACKAGE_BUILDER_GLIBC} AS node-package-builder-glibc
+FROM centos/devtoolset-7-toolchain-centos7${NODE_PACKAGE_BUILDER_GLIBC} AS node-package-builder-glibc
+USER 0
 WORKDIR /tmp
 COPY scripts/node_builder_glibc_env.sh .
 RUN ./node_builder_glibc_env.sh
 COPY scripts/build_node_package.sh .
 RUN ./build_node_package.sh
+USER 1001
```

Comment:
Not required because the next command is `FROM` so we switch image.

Reply:
That's what I thought too but linter failed: https://github.com/Granulate/gprofiler/actions/runs/3514434057/jobs/5888441878